org/contributors.yml

```diff
@@ -633,7 +627,6 @@ contributors:
 - xtreme-conor-nosal
 - xtreme-debbie-chen
 - xtreme-jason-smith
-- xtreme-jesse-malone
```

Comment:
I think Jesse did eventually accept an invitation, but he hasn't contributed to CF repositories in at least a couple of years, so any reasonable contribution policy would recommend his removal anyway.